lib/usecases/getSecureUploadUrl.test.js

```diff
@@ -0,0 +1,51 @@
+import getSecureUploadUrl from './getSecureUploadUrl';
+
+jest.mock('../gateways/s3', () => () => ({
+  createUploadUrl: async () => {
+    return {
+      url: 'https://s3.amazonaws.com/document-id',
+      fields: {},
+    };
+  },
+}));
+
+describe('get secure upload URL usecase', () => {
+  let initialEnv;
+  beforeEach(() => {
+    jest.resetAllMocks();
+
+    initialEnv = {
+      ...process.env,
+    };
+  });
+
+  afterEach(() => {
+    process.env = initialEnv;
+  });
+
+  it('should use local filesystem gateway if the ENV environment variable is "local"', async () => {
+    process.env.ENV = 'local';
```

Comment:
Might be worth adding ENV to the Readme. I've added ENV=local to my environment. This PR saves commenting out the code to test each time, nice!